tests/longhaul/block_test.go

```diff
@@ -24,7 +24,9 @@ func TestK8sBlockLongHaul(t *testing.T) {
 }
 
 var (
-	logger = capnslog.NewPackageLogger("github.com/rook/rook", "longhaul")
+	logger               = capnslog.NewPackageLogger("github.com/rook/rook", "longhaul")
+	defaultNamespace     = "default"
+	defaultRookNamespace = "test-rook"
```

Comment:
if you're going to redefine this const, i'd suggest a different namespace such as `longhaul`

Reply:
updated.